### PR TITLE
[9.0][MIG][website_snippet_calendar] 

### PR DESCRIPTION
**Pull request by tarteo**
Previously (branch 8.0) the name was 'website_calendar_snippet'

### CODE REVIEW

**Comment by tarteo** [comment]
Tests are working locally but not here (same issue as in https://github.com/OCA/web/pull/548)

File: website_snippet_calendar/tests/test_website_snippet_calendar.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        event_obj = self.env['calendar.event']
+        event_obj.create({
+            'name': 'Test',
+            'start': '2017-03-01 12:00:00',
+            'stop': '2017-03-01 13:00:00'
+        })
+        events = event_obj.search([('start', '<', select_stop),
+                                   ('stop', '>', select_start)])
+        request = http.request
+        http.request = MagicMock(env=self.env,
```
Comment (simahawk) [inline website_snippet_calendar/tests/test_website_snippet_calendar.py]: check Dave's comment in OCA/web/pull/548. You should use a context manager.

File: website_snippet_calendar/static/src/js/website_snippet_calendar.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+                        all_filters[-1] = get_color(-1);
+                        _.each(result.contacts, function (c) {
+                            if (!all_filters[c]) {
+                                all_filters[c] = get_color(c);
+                            }
+                        });
+
+                        //Mutate data for calendar
+                        var events = result.events;
+                        for(var i = 0; i < events.length; i++) {
```
Comment (simahawk) [inline website_snippet_calendar/static/src/js/website_snippet_calendar.js]: what about moving this to specific function? Something like `prepare_events` would ease customization.

File: website_snippet_calendar/README.rst [hunk 3]
```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+Calendar Snippet
+================
+
+This module adds a new block in the website editor.
+The new block is based on Calendar (module). Use field 'Privacy'
```
Comment (simahawk) [inline website_snippet_calendar/README.rst]: do you mean "FullCalendar"? :)

**Review by simahawk — CHANGES_REQUESTED** [review]

File: website_snippet_calendar/README.rst (hunk 3 above)
Comment (tarteo) [reply website_snippet_calendar/README.rst]: No, I mean the calendar of Odoo module 'Calendar'.

File: website_snippet_calendar/README.rst [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/186/9.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/186/issues>`_. In case of trouble, please
```
Comment (astirpe) [inline website_snippet_calendar/README.rst]: Replace `186` with `website`

**Review by astirpe — APPROVED** [review]
Only a minor fix in the readme, other than that it LGTM

**Comment by astirpe** [comment]
Branch rebased

**Pull request closed without merging**